presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -141,6 +141,10 @@ String Functions
     Returns the starting position of the first instance of ``substring`` in
     ``string``. Positions start with ``1``. If not found, ``0`` is returned.
 
+.. function:: starts_with(string, substring) -> boolean
+
+    Tests whether substring is a prefix of string".
```

Comment:
substring and string should be surrounded by `

Also, there's an extra " at the end of the sentence.